Sources/ClientRuntime/Networking/UnknownServiceError.swift

```diff
@@ -10,10 +10,13 @@ public struct UnknownServiceError: ServiceError, Equatable {
     public var _isThrottling: Bool = false
     public var _type: ErrorType = .unknown
     public var _message: String?
+    public var _errorType: String?
```

Comment:
Can we add a comment describing what error type is here?

Reply:
Added